oak_functions/examples/weather_lookup/README.md

```diff
@@ -57,6 +57,42 @@ radius of 40km. See https://s2geometry.io/devguide/examples/coverings for more
 information on coverings. Cell coverings for areas can be visualised using
 https://s2.sidewalklabs.com/regioncoverer/.
 
+### File format
+
+The weather file format looks as follows:
+
+Weather file is a Protobuf compiled file that consists of a list of the
+following entries:
+
+```protobuf
+message Entry {
+  bytes key = 1;
+  bytes value = 2;
+}
+```
+
+There are 2 types of entries:
+
+- Location entries
+- Cell entries
+
+_Location_ entries are represented as:
+
+- `key` is `LATITUDE_SIGNED_BIG_ENDIAN | LONGITUDE_SIGNED_BIG_ENDIAN`
+  [Length-Delimited](https://developers.google.com/protocol-buffers/docs/encoding#structure)
+  encoded
+  - Where `|` is concatenation
+  - Both Latitude and Longitude are in _microdegrees_
+- `value` is `bytes` that correspond to weather
+
+Each _Location_ entry has a single corresponding _Cell_ entry
```

Comment:
This is not a 1-to-1 mapping between location entries and index entries. There is a many-to-many mapping. There is an index entry for every S2 geometry cell that has at least one weather location in its vicinity (within 40 km of any point in the cell). So a single index entry can contain multiple keys, and a single key can be present in multiple index entries.

Reply:
Done